src/containers/About.jsx

```diff
@@ -28,66 +28,60 @@ class About extends Component {
             <Container className="mt-4 pl-0">
               <h1>Meet The Team</h1>
               <p>
-                B-Soc represents the odd years (SE2021, SE2023, SE2025) and
-                A-Soc represents the even years (SE2022, SE2024). We’re a
+                A-Soc represents the even years (SE2022, SE2024, SE2026) and
+                B-Soc represents the odd years (SE2023, SE2025). We’re a
                 friendly bunch, feel free to say hi if we ever bump into each
                 other online or on campus!
               </p>
               <ProfilePhotoGroup
-                soc={'B-Soc'}
+                soc={'A-Soc'}
                 execs={[
                   {
-                    name: 'Atif Mahmud',
-                    src: profileImage('atif'),
-                    year: '2023',
+                    name: 'Aaron Abraham',
+                    src: profileImage('aaron'),
+                    year: '2024',
                     position: 'President',
                   },
                   {
-                    name: 'Elisa Luan',
-                    src: profileImage('elisa'),
-                    year: '2023',
-                    position: 'VP Events',
+                    name: 'Yash Arora',
+                    src: profileImage('yash'),
+                    year: '2024',
+                    position: 'VP Communications',
                   },
                   {
-                    name: 'Mayank Kanoria',
-                    src: profileImage('mayank'),
-                    year: '2023',
-                    position: 'VP Finance',
+                    name: 'Danny Wu',
+                    src: profileImage('danny'),
+                    year: '2024',
+                    position: 'VP Events',
                   },
                 ]}
                 reps={[]}
-                color={'var(--color-math)'}
+                color={'var(--color-eng)'}
```

Comment:
keep the colours the same for visual consistency.

The current About page has `color-math` then `color-eng` and then the contact us bottom bar is also `color-math`. 
![image](https://user-images.githubusercontent.com/14525856/147140155-94333671-58d6-4d42-8621-d46b447f2b59.png)

A little history: You may also notice that the purple and pink colours alternate as you go between different tabs (i.e. Home, Events, Blog are purple and Events, Blog, Wise are pink). 
![image](https://user-images.githubusercontent.com/14525856/147140336-fd0783ff-0a72-4066-aa57-1f629cc40801.png)

Feel free to switch it up if you'd like to do something else, just keep it consistent :)